packages/account/src/Constants/routes-config.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import Loadable from 'react-loadable';
+import { Loading } from '@deriv/components';
+import { addRoutesConfig } from '@deriv/shared/utils/route';
+import { localize } from '@deriv/translations';
+// TODO remove this once deriv-shared is updated
+import routes from './routes';
+
+// Error Routes
+const Page404 = React.lazy(() => import(/* webpackChunkName: "404" */ 'Modules/Page404'));
+
+const handleLoading = (props) => {
+    // 200ms default
+    if (props.pastDelay) {
+        return <Loading />;
+    }
+    return null;
+};
+
+const makeLazyLoader = (importFn) => (component_name) =>
+    Loadable.Map({
+        loader: {
+            ComponentModule: importFn,
+        },
+        render(loaded, props) {
+            const ComponentLazy = loaded.ComponentModule.default[component_name];
+            return <ComponentLazy {...props} />;
+        },
+        loading: handleLoading,
+    });
+
+const lazyLoadAccountComponent = makeLazyLoader(() => import(/* webpackChunkName: "account-sections" */ 'Sections'));
```

Comment:
The Account module is now already lazy loaded into the main application (Core), we can make all of these sync, or at least remove the lazy loading here of the main `routes.account` path as it now creates 2 lazy steps.